pkg/storage/pebble.go

```diff
@@ -1078,32 +1078,43 @@ func NewPebble(ctx context.Context, cfg PebbleConfig) (p *Pebble, err error) {
 
 	db, err := pebble.Open(cfg.StorageConfig.Dir, opts)
 	if err != nil {
+		if !minVerFileExists && strings.Contains(err.Error(), "already exists") {
```

Comment:
`strings.Contains`? Wouldn't we want a structured error here? An argument could be made that it isn't worth it because it's such a dead code path & the event of a false positive isn't dramatic since the startup was failing either way?

Reply:
Yeah I would add a better way in Pebble. But for now I'm not sure of the higher level logic - what are the expectations if a machine fails during bootstrap? If it fails after creating the store but before creating the min version file, we would need manual cleanup before trying to bootstrap the store again. OTOH if we allow an existing store, then we might open an old version. Maybe the right thing is to add an ErrorIfNotEmpty mode to Pebble?

Reply:
Done.